packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.tsx

```diff
@@ -322,7 +322,12 @@ export default function DownloadListTable(props: DownloadListTableProps) {
                               associatedObjectType={
                                 FileHandleAssociateType.FileEntity
                               }
-                              entityVersionNumber={item.versionNumber.toString()}
+                              {...(item.versionNumber !== null
+                                ? {
+                                    entityVersionNumber:
+                                      item.versionNumber.toString(),
+                                  }
+                                : {})}
```

Comment:
Can we just do something like this?
```suggestion
                              entityVersionNumber={item.versionNumber?.toString()}
```

Reply:
Thanks for the suggestion. Yes, we could definitely do that.